docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -150,7 +150,79 @@ You should see the URL for your bootstrap package as the value for `mdm.macos_se
 
 ## macOS Setup Assistant
 
-> This feature is currently in development.
+When an end user unboxes their new Mac, or starts up a freshly wiped Mac, they're presented with the macOS Setup Assistant. Here they see panes that allow them to configure accessibility, appearance, and more.
+
+In Fleet, you can customize the macOS Setup Assistant by using an automatic enrollment profile.
+
+To customize the macOS Setup Assistant, we will do the following steps:
+
+1. Create an automatic enrollment profile
+2. Upload the profile to Fleet
+3. Test the custom macOS Setup Assistant
+
+### Step 1: create an automatic enrollment profile
+
+1. Download Fleet's example automatic enrollment profile by navigating to the example [here on GitHub](https://github.com/fleetdm/fleet/blob/main/mdm_profiles/setup_assistant.json) and clicking the download icon.
+
+2. Open the automatic enrollment profile and replace the `profile_name` key with your organization's name.
+
+3. View the the list of macOS Setup Assistant properties (panes) [here in Apple's Device Management documentation](https://developer.apple.com/documentation/devicemanagement/skipkeys) and choose which panes to hide from your end users.
+
+4. In your automatic enrollment profile, edit the `skip_setup_items` array so that it includes the panes you want to hide.
+
+### Step 2: upload the profile to Fleet
+
+Choose which team you want to add the automatic enrollment profile to.
+
+In this example, we'll add the automatic enrollment profile to the "Workstations (canary)" team. Only hosts that automatically enroll to this team will see the custom macOS Setup Assistant.
+
+2. Create a `workstations-canary-config.yaml` file:
+
+```yaml
+apiVersion: v1
+kind: team
+spec:
+  team:
+    name: Workstations (canary)
+    mdm:
+      macos_setup:
+        macos_setup_assistant: ./path/to/automatic_enrollment_profile.json
```

Comment:
why is the key here `macos_setup_assistant` but the content is the automatic enrollment profile? I would think the automatic enrollment profile has more settings than just the setup assistant? Or should the key just be named automatic_enrollment_profile?

Reply:
> the automatic enrollment profile has more settings than just the setup assistant

This is right. For example, IT admin can edit `is_mdm_removable` and other settings documented [here](https://developer.apple.com/documentation/devicemanagement/profile).

We went with the `macos_setup_assistant` name in the config (and UI) because we learned that the majority of use cases for the auto enrollment profile involve showing/hiding the setup assistant panes.

That said, the automatic enrollment profile has more settings and our docs should clarify this for users that are looking to modify settings outside of the setup assistant. I added a note to the docs here: https://github.com/fleetdm/fleet/pull/12812/files#diff-53f79a9bcbecfe28f586509c8c8012fc032b247860f51dcbc0c0254d1644a0b2R173